### ISSUE TITLE
Production Use?

### ISSUE THREAD

**Issue by EisenbergEffect** [torrmal/cordova-standalone-hydration#1]
I've been looking around to see if anyone had done something like this. I was wondering if this was safe for production use, or only suitable for testing. Can I put an app in the Apple AppStore with this updater in it? or will that be rejected? Have you tried something like this?

Also, I find this interesting as a technique to load plugins to an app. The idea being that the app wouldn't auto-update, but that certain functionality might be dynamically plugged in depending on what the user is doing. Would this technique be suitable for that? Would there be a problem with any of the app stores?

Thanks!

**Comment by lylepratt** [comment]
Supposedly Apple allows this: https://www.meteor.com/blog/2014/10/02/Apple-hot-code-push-mobile

Since I saw that stuff last week about meteor using this in production, I've been wanting to do the same with my Cordova apps. I believe I'm going to give it a shot. 

@torrmal Would you mind tell us a little about your experience using this and what you wrote it for?

**Comment by driverdan** [comment]
I wouldn't consider this code suitable for production. The MD5 code, for example has the comment `MD5: Cant remember where i got this one from` which means the quality of it and license are unknown.

**Comment by lylepratt** [comment]
I agree. However, there are many MD5 javascript libraries out there that are MIT licensed:
https://github.com/blueimp/JavaScript-MD5

The code is straightforward. May just do something similar with an Angular module.

**Comment by hems** [comment]
@driverdan lol nice joke!
